tools/dissector/packet-opendds.cpp

```diff
@@ -290,8 +290,9 @@ namespace OpenDDS
       offset += len;
 
       // hf_sequence
-      size_t size = 0, padding = 0;
-      gen_find_size(header.sequence_, size, padding);
+      size_t size = 0;
+      const Encoding encoding(Encoding::KIND_CDR_UNALIGNED);
+      serialized_size(encoding, size, header.sequence_);
```

Comment:
```suggestion
      size_t size = serialized_size(encoding, header.sequence_);
```

You can use this when the size starts at zero.